src/app/_components/Sort.tsx

```diff
@@ -40,6 +42,7 @@ export function Sort({ query, options, defaultQuery }: SortProps) {
     <SortListbox
       options={options}
       selected={selectedOption}
+      icon={ArrowsDownUp}
```

Comment:
Same comments here - `buttonIcon` may be clearer - And We could have `ArrowsDownUp` as the default?